content/python/encyclopedia/classes-in-python.md

```diff
@@ -0,0 +1,47 @@
+<!--
+---
+Subjects:
+  - "What are Classes in Python?"
+Tags: 
+  - "python"
+  - "classes"
+  - "object oriented"
+---
```

Comment:
Please add the Catalog Content field, [as in the template](https://github.com/Codecademy/codepedia-content/blob/main/documentation/encyclopedia-entry-template.md)